Sources/StreamChatUI/Common Views/ChatAvatarView.swift

```diff
@@ -7,11 +7,7 @@ import UIKit
 open class ChatAvatarView: View {
     // MARK: - Subviews
     
-    public let imageView: UIImageView = {
-        let imageView = UIImageView()
-        imageView.translatesAutoresizingMaskIntoConstraints = false
-        return imageView
-    }()
+    open var imageView: UIImageView = UIImageView().withoutAutoresizingMaskConstraints
```

Comment:
👍 for making this open, but to make sure it still works correctly, shouldn't we make it `private(set)`? Aka - you can provide your custom object in subclasses but you can't modify it during the runtime.

Reply:
Thanks, you are right :) Changed.